network/junos/junos_command.py

```diff
@@ -183,9 +183,9 @@ def parse(module, command_type):
             item = dict(command=item, output=None)
         elif 'command' not in item:
             module.fail_json(msg='command keyword argument is required')
-        elif item.get('output') not in [None, 'text', 'xml']:
+        elif item.get('output') not in [None, 'text', 'xml', 'json']:
```

Comment:
I know this isn't @stianvi code, but why are we doing this check, rather than just trusting `choices=['text', 'xml', 'json'],`?